tests/Command/AbstractCommandTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Tests\Command;
+
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+
+abstract class AbstractCommandTest extends KernelTestCase
+{
+    protected function setUp(): void
+    {
+        exec('stty 2>&1', $output, $exitcode);
```

Comment:
I was checking the `Terminal::hasSttyAvailable()` method:

https://github.com/symfony/symfony/blob/de39dbae8f2425f48e790c4f1b3e70f9b702554f/src/Symfony/Component/Console/Terminal.php#L63

It's public ... but marked as `@internal`, so I guess we can't use it. So, could we please at least borrow the check for the tricky `exec()` function to not fail in sistems that have disabled it? Thanks!

Reply:
This is a refactoring (moving code by copy/pasting) from the already existing test. So it seems out of my scope. But I can add it anyway.

Do you mean adding such code?

```php
<?php
        if (!\function_exists('exec')) {
            $this->skip();
        }
```

Reply:
It's true that it's not strictly related to your changes, so sorry for asking this ... but don't you think it would make the code slightly better? The `exec()` function is so special that many servers disable it. Thanks!

Reply:
You didn't answer my question but well, I'll consider it as a yes ^^ .